dataland-frontend/src/components/forms/parts/fields/YesNoFormField.vue

```diff
@@ -22,7 +22,77 @@
         <FormKit type="text" name="reference" v-model="documentReference" :outer-class="{ 'hidden-input': true }" />
       </FormKit>
     </FormKit>
+    <div v-else-if="evidenceDesired">
+      <FormKit v-model="baseDataPointYesNo" type="group" :name="name">
+        <div class="mb-3">
+          <RadioButtonsFormElement
+            name="value"
+            v-model="currentValue"
+            :validation="validation"
+            :validation-label="validationLabel ?? label"
+            :options="yesNoOptions"
+            :data-test="dataTest"
+          />
+        </div>
+        <div>
+          <FormKit type="group" name="dataSource">
+            <div class="next-to-each-other">
+              <div class="flex-1">
+                <UploadFormHeader :label="`${label} Report`" :description="'Upload Report'" />
+                <FormKit
+                  type="select"
+                  name="report"
+                  v-model="currentReportValue"
+                  placeholder="Select a report"
+                  :options="['None...', ...injectReportsName]"
+                />
+              </div>
+              <div>
+                <UploadFormHeader :label="'Page'" :description="'Page where information was found'" />
+                <FormKit
+                  outer-class="w-100"
+                  v-model="currentPageValue"
+                  type="number"
+                  name="page"
+                  placeholder="Page"
+                  validation-label="Page"
+                  step="1"
+                  min="0"
+                  validation="min:0"
+                />
+              </div>
+            </div>
+          </FormKit>
+        </div>
 
+        <!-- Data quality -->
+        <div class="mb-4">
+          <UploadFormHeader
+            :label="`${label} Quality`"
+            description="The level of confidence associated to the value."
+            :is-required="isDataQualityRequired"
+          />
+          <div class="md:col-6 col-12 p-0">
+            <FormKit
+              type="select"
+              :modelValue="!isDataQualityRequired ? 'NA' : currentQualityValue"
```

Comment:
The fallback could be simply an empty string `''`

Reply:
Changed

Reply:
👍